app/presenters/pluralize.rb

```diff
@@ -0,0 +1,14 @@
+class Pluralize
```

Comment:
I don't think so, we're re-using ActionView's `pluralize` which I thin kis exactly what we want (pass in number and word and get pluralized or non-plurialized result depending) http://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize